src/test/java/core/basesyntax/service/fileservice/FileReaderImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax.service.fileservice;
+
+import static junit.framework.TestCase.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileReaderImplTest {
+    private static final String EMPTY_FILE = "emptyFile.csv";
+    private static final String EMPTY_NAME = "";
+    private static final String OTHER_FORMAT = "txtFileFormat.txt";
+    private static final String MISSING_FILE = "missingFile.csv";
+    private static final String VALID_FILE = "validFile.csv";
+    private static List<String> validOutputData;
+    private static FileReader reader;
+
+    private static File emptyFile;
+    private static File validFile;
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        reader = new FileReaderImpl();
+        validOutputData = new ArrayList<>();
+        emptyFile = new File(EMPTY_FILE);
```

Comment:
You do not need to create file using `new File(EMPTY_FILE)`
please create `emptyFile.csv` file in resources
Same for `validFile.csv`